glide.yaml

```diff
@@ -34,6 +34,7 @@ import:
   - common/websocket
   - stream
 - package: github.com/uber/cherami-thrift
+  version: output_admin
```

Comment:
TODO: will fix this once the corresponding diff on the cherami-thrift repo is landed.